client/src/containers/home/StatisticFragment.js

```diff
@@ -1,17 +1,35 @@
 import React from 'react';
 import { Row, Col, Card } from 'antd';
-import { Trans } from '@lingui/react';
+import { Trans } from '@lingui/macro';
+
+import { ReactComponent as PeopleRisk } from '../../images/user-solid.svg';
+import { ReactComponent as EvaluatedBuildings } from '../../images/home-solid.svg';
+import { ReactComponent as ConsolidatedBuildings } from '../../images/house-damage-solid.svg';
 
 import config from '../../config';
 
 const { STATISTICS_URL } = config;
 
-const StatisticCard = ({ children, value }) => (
-  <Card style={{ border: 'none', backgroundColor: '#ee741b' }}>
-    <div>
-      <h1 style={{ color: 'white' }}>{value}</h1>
-      <h3 style={{ color: 'white' }}>{children}</h3>
-    </div>
+const StatisticCard = ({ title, value, icon }) => (
+  <Card style={{ border: 'none', backgroundColor: '#ee741b', color: 'white' }} size="small">
+    <Row type="flex" justify="space-between" align="middle">
+      <Col span={4}>{icon}</Col>
+      <Col span={18} style={{ textAlign: 'left' }}>
+        {value > 0 ? (
+          <>
+            <h3 style={{ color: '#fff' }}>{value}</h3>
```

Comment:
Couldn't the color style be applied to the parent ?

Reply:
No, but I moved the styling to the `common.scss` file to avoid all that repetition.